aws/resource_aws_resource_group_test.go

```diff
@@ -0,0 +1,103 @@
+package aws
+
+import (
+	"fmt"
+	"testing"
+
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/resourcegroups"
+	"github.com/hashicorp/terraform/helper/acctest"
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+)
+
+func TestAccAWSResourceGroup_importBasic(t *testing.T) {
+	resourceName := "aws_resource_group.test"
+	n := fmt.Sprintf("test-group-%d", acctest.RandInt())
+
+	resource.ParallelTest(t, resource.TestCase{
+		PreCheck:  func() { testAccPreCheck(t) },
+		Providers: testAccProviders,
+		Steps: []resource.TestStep{
+			{
+				Config: testAccAWSResourceGroupConfig_basic(n),
+			},
+
+			{
+				ResourceName:      resourceName,
+				ImportState:       true,
+				ImportStateVerify: true,
+			},
+		},
+	})
+}
+
+func TestAccAWSResourceGroup_basic(t *testing.T) {
+	resourceName := "aws_resource_group.test"
+	n := fmt.Sprintf("test-group-%d", acctest.RandInt())
```

Comment:
We have a provider SDK helper function for this 😄 

```suggestion
	n := acctest.RandomWithPrefix("tf-acc-test")
```